v20.2/follower-reads.md

```diff
@@ -65,14 +65,14 @@ To verify that your cluster is performing follower reads:
 
 ### Run queries that use follower reads
 
-<a name="experimental-follower-read-timestamp"></a>
+<a name="follower-read-timestamp"></a>
```

Comment:
FWIW this anchor can be deleted, nothing is pointing to it (I was just making this exact PR until I saw this lol)

Reply:
Ah! I see - sorry to step on your toes with this PR!

Reply:
You did not - in fact I'm thrilled - thanks for doing this

Reply:
NP! Just removed that anchor.